geo/circle.go

```diff
@@ -0,0 +1,37 @@
+package geo
+
+import (
+	"encoding/json"
+)
+
+// Circle specified by a center point and radius with units,
+// which default to meters.
+type Circle struct {
+	Type        string    `json:"type"`
+	Radius      string    `json:"radius"`
+	Coordinates []float64 `json:"coordinates"`
+}
+
+// NewCircle creates and initializes a new Circle.
```

Comment:
is there a method to verify the arguments? 
for example:  the order of coordinates, Lon and Lat or Lat and Lon.

Reply:
Good point, no, there's not, I will implement it.

Reply:
There are some functions like this in [`geo_point.go`](https://github.com/olivere/elastic/blob/release-branch.v6/geo_point.go).

Reply:
They are roughly based on [`GeoPoint.java`](https://github.com/elastic/elasticsearch/blob/6.7/server/src/main/java/org/elasticsearch/common/geo/GeoPoint.java).